qcodes/tests/test_station.py

```diff
@@ -0,0 +1,203 @@
+import pytest
+
+from qcodes import Instrument
+from qcodes.station import Station
+from qcodes.tests.instrument_mocks import DummyInstrument
+from qcodes.tests.test_combined_par import DumyPar
+from qcodes.instrument.parameter import Parameter
+
+
+@pytest.fixture(autouse=True)
+def set_default_station_to_none():
+    """Makes sure that after startup and teardown there is no default station"""
+    Station.default = None
+    yield
+    Station.default = None
+
+
+@pytest.fixture(autouse=True)
+def close_all_instruments():
+    """Makes sure that after startup and teardown all instruments are closed"""
+    Instrument.close_all()
+    yield
+    Instrument.close_all()
+
+
+def test_station():
+    bob = DummyInstrument('bob', gates=['one'])
+    station = Station(bob)
+
+    assert ['bob'] == list(station.components.keys())
+    assert bob == station.components['bob']
+
+    assert station == station.default
+    assert station == Station.default
```

Comment:
first is checking the instance's `default`, the second checks the class's `default`, to confirm that they are the same. it's really just a "just in case" assertion :)